tests/test_tapioca.py

```diff
@@ -220,6 +221,34 @@ def test_post_request(self):
         self.assertEqual(response().data, {'data': {'key': 'value'}})
 
     @responses.activate
+    def test_token_expired_and_not_refresh_flag(self):
+        responses.add(responses.POST, self.wrapper.test().data,
+                      body='{"error": "Token expired"}',
+                      status=401,
+                      content_type='application/json')
+        with self.assertRaises(ClientError) as context:
+            response = self.wrapper.test().post()
+
+    @responses.activate
+    def test_token_expired_and_refresh_flag(self):
+        self.first_call = True
+        responses.add_callback(
+          responses.POST, self.wrapper.test().data,
+          callback=self.request_callback,
+          content_type='application/json',
+        )
+        
+        response = self.wrapper.test().post(True)
```

Comment:
Lets be explicit when passing parameters:
``response = self.wrapper.test().post(refresh_auth=True)``